lib/dal/src/func/authoring.rs

```diff
@@ -466,6 +584,43 @@ impl FuncAuthoringClient {
         Ok(())
     }
 
+    #[instrument(level = "info", name = "func.authoring.save_code", skip(ctx))]
+    /// Save only the code for the given [`FuncId`]
+    pub async fn save_code(
```

Comment:
I wonder if separate save code from save metadata is the right approach here - right now the way our system works you can get a conflict if you run those two requests in parallel, which can be easily done via the UI

Those kinds of races caused some problems for asset code recently

Reply:
yeah we'll feel our way through this - once you can no longer conflict with yourself it'll be less painful